com.unity.ml-agents/Runtime/Sensors/RayPerceptionSensorComponent2D.cs

```diff
@@ -14,7 +14,7 @@ public class RayPerceptionSensorComponent2D : RayPerceptionSensorComponentBase
         public RayPerceptionSensorComponent2D()
         {
             // Set to the 2D defaults (just in case they ever diverge).
-            RayLayerMask = Physics2D.DefaultRaycastLayers;
+            RayLayerMask = -5;
```

Comment:
Don't need this anymore, now that we're always using `k_PhysicsDefaultLayers` in the base class. (sorry, github wouldn't let me do a multi-line suggestion).